layouts/taxonomy/author.html

```diff
@@ -1,56 +1,53 @@
 {{ define "main" }}
-    <div class="md:flex">
+    <div class="container mx-auto px-8 py-8 md:px-0">
+        <div class="md:flex">
 
-        <div class="md:w-3/12 pr-8">
-            {{ partial "blog/sidebar.html" . }}
-        </div>
+            <div class="md:w-3/12 pr-8">
+                {{ partial "blog/sidebar.html" . }}
+            </div>
 
-        <div class="md:w-7/12">
-            {{ $authorData := index $.Site.Data.team.team .Data.Term }}
-            {{ $paginator := .Paginate .Data.Pages.Reverse }}
+            <div class="md:w-7/12">
+                {{ $authorData := index $.Site.Data.team.team .Data.Term }}
+                {{ $paginator := .Paginate .Data.Pages.Reverse }}
 
-            <header>
-                <div class="flex items-top">
-                    <span class="rounded-full bg-gray-300 mr-8">
-                        <img class="rounded-full h-32 p-1" src="/images/team/{{ $authorData.id }}.jpg" alt="{{ $authorData.name }}">
-                    </span>
-                    <div>
-                        <h1 class="no-anchor">{{ $authorData.name }}</h1>
+                <header>
+                    <div class="flex items-top">
+                        <span class="rounded-full bg-gray-300 mr-8">
+                            <img class="rounded-full h-32 p-1" src="{{ $.Site.BaseURL }}images/team/{{ $authorData.id }}.jpg" alt="{{ $authorData.name }}">
```

Comment:
I think this can be reverted back to the change I made to remove `{{ $.Site.BaseURL }}`. (Sorry for the conflict!)

```suggestion
                            <img class="rounded-full h-32 p-1" src="/images/team/{{ $authorData.id }}.jpg" alt="{{ $authorData.name }}">
```